_search-plugins/knn/knn-index.md

```diff
@@ -116,7 +116,7 @@ Lucene HNSW implementation ignores `ef_search`  and dynamically sets it to the v
 ### Supported faiss encoders
 
 You can use encoders to reduce the memory footprint of a k-NN index at the expense of search accuracy. faiss has
-several encoder types, but the plugin currently only supports *flat* and *pq* encoding.
+several encoder types, but the plugin currently only supports *flat*, *pq* and *SQfp16* encoding.
```

Comment:
```suggestion
several encoder types, but the plugin currently only supports *flat*, *pq*, and *SQfp16* encoding.
```

Reply:
Updated

Reply:
@naveentatikonda we arent going with SQfp16 as the name right?